frontend/src/old-pages/Clusters/JobSubmitDialog.tsx

```diff
@@ -10,9 +10,13 @@
 // limitations under the License.
```

Comment:
suggestion(non-blocking): there is a bit of logic in this component not covered by tests. It can be added in another PR since this one already got a lot of iterations